hack/build/run-functional-tests.sh

```diff
@@ -58,4 +81,4 @@ velero_path=$(pwd)/${VELERO_DIR}
 (
     cd ${TESTS_DIR}
     KUBECONFIG=${kubeconfig_arg} PATH=${PATH}:${velero_path} ${test_command}
-)
```

Comment:
no newline?

Reply:
its a redundant new line at the end of the file..

Reply:
Empty newline is the standard across our repos AFAIK